src/Playwright/Core/BrowserType.cs

```diff
@@ -43,19 +44,19 @@ internal BrowserType(ChannelOwner parent, string guid, BrowserTypeInitializer in
         _initializer = initializer;
     }
 
-    internal PlaywrightImpl Playwright { get; set; }
+    internal PlaywrightImpl Playwright { get; set; } = null!;
 
     public string ExecutablePath => _initializer.ExecutablePath;
 
     public string Name => _initializer.Name;
 
     [MethodImpl(MethodImplOptions.NoInlining)]
-    public async Task<IBrowser> LaunchAsync(BrowserTypeLaunchOptions options = default)
+    public async Task<IBrowser> LaunchAsync(BrowserTypeLaunchOptions? options = default)
```

Comment:
Will this break older version .net users? Or do we not care about that?

Reply:
This is not the public interface, the public interface is [this](https://github.com/microsoft/playwright-dotnet/blob/8c6b30bb769908065c09e54d8dd18d6a0b9a93d3/src/Playwright/API/Generated/IBrowserType.cs#L160) which is nullable.